README.md

```diff
@@ -65,6 +65,57 @@ widely used, while draft-03 is long since deprecated.
 If you see anything missing from the current supported drafts, or incorrect
 on any draft still accepting bug fixes, please file an issue or submit a PR.
 
+Optional Tests
+--------------
+
+Tests in the `optional/` directory test optional validation behavior. Validators
+only need to execute tests for implemented features.
+
+There are different sets of optional features for each draft. They are
+summarized below.
+
+1. Arbitrary-precision number support: JSON allows numbers of
+   arbitrary precision.\
+   See: [6. Numbers](https://tools.ietf.org/html/rfc8259#section-6)\
+   Also see:
+   * 2019-09: [4.2. Validation of Numeric Instances](https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.4.2)
```

Comment:
It's going to be a bit of a pain in the butt to maintain this as new versions come out (since each time that happens we'll have to go find the relevant new section in that draft).

But such is life I suppose, I don't have a better suggestion.